src/siteQualityMapping.js

```diff
@@ -46,10 +46,12 @@ chrome.contextMenus.onClicked.addListener(async (info, tab) => {
 
 
 chrome.tabs.onUpdated.addListener(async (tabId, changeInfo, tab) => {
+	console.log('onUpdate');
```

Comment:
Don’t push debugging logs

Reply:
Hi TheENGR,

This has been resolved as I have removed the console.log lines.